.github/workflows/playwright.yml

```diff
@@ -352,8 +364,20 @@ jobs:
       if: ${{ !cancelled() && (success() || failure()) }}
       continue-on-error: true
       with:
-        name: test-results-${{ matrix.os }}-${{ github.sha }}
+        name: test-results-electron-${{ matrix.os }}-${{ github.sha }}
```

Comment:
This was definitely a bug, but I don't know why the non-electron tests were failing